server/vulnerabilities/oval/mappers.go

```diff
@@ -89,12 +89,16 @@ func mapPackageState(sta oval_input.DpkgStateXML) ([]oval_parsed.ObjectStateEvrS
 }
 
 func mapPackageObject(obj oval_input.DpkgObjectXML, vars map[string]oval_input.ConstantVariableXML) ([]string, error) {
+	if obj.Name.Value != "" {
```

Comment:
Sorry to insist on documentation, let's add a comment explaining the why (even mention Ubuntu 19 is the one with this issue). Mostly for future readers of this code.

Reply:
Thanks!